moveit_core/kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -37,12 +37,11 @@
 #ifndef MOVEIT_KINEMATICS_BASE_KINEMATICS_BASE_
 #define MOVEIT_KINEMATICS_BASE_KINEMATICS_BASE_
 
-#include <geometry_msgs/PoseStamped.h>
-#include <moveit_msgs/MoveItErrorCodes.h>
+#include <geometry_msgs/msg/pose_stamped.hpp>
+#include <moveit_msgs/msg/move_it_error_codes.hpp>
```

Comment:
``move_it`` is always treated as one word: ``moveit``. for example, the string right before it says ``moveit_msgs`` not ``move_it_msgs``

Reply:
@davetcoleman, can't do that. This is imposed by the transformation from .msgs to header files and I doubt @wjwwood and other would accept a change in the core ros2 libraries only for this. Simplest way of doing what you suggest would require renaming `MoveItErrorCodes.msg` into `MoveitErrorCodes.msg`. I'm ok with doing so but that's outside of the scope of this PR.

Reply:
I was never really in favor of the implicit transformation myself, but that is what's there now and there was a long discussion about it at the time (so it wasn't picked arbitrarily). So I agree that it would be difficult to decide to change it now.

Perhaps a more likely proposal would be to generate both patterns (e.g. `move_it_error_codes.hpp` and `MoveItErrorCodes.hpp`), so you could use which ever you like.

I will say though, just in my opinion, that `MoveitErrorCodes.msg` looks more correct to me than `MoveItErrorCodes.msg`, though that's completely subjective.

Reply:
I agree we should probably just rename the message to `MoveitErrorCodes.msg`.

I raised an issue in our fork at https://github.com/AcutronicRobotics/moveit_msgs/issues/4. I'm resolving this comment for now.